ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/photo/PhotoTaskFragment.kt

```diff
@@ -63,8 +63,16 @@ class PhotoTaskFragment : AbstractTaskFragment<PhotoTaskViewModel>() {
   @Inject lateinit var popups: EphemeralPopups
   @Inject lateinit var navigator: Navigator
 
-  private lateinit var selectPhotoLauncher: ActivityResultLauncher<String>
-  private lateinit var capturePhotoLauncher: ActivityResultLauncher<Uri>
+  private var selectPhotoLauncher: ActivityResultLauncher<String> =
+    registerForActivityResult(ActivityResultContracts.GetContent()) { uri: Uri? ->
```

Comment:
`registerForActivityResult` is neat! We can probably get rid of `ActivityStreams` altogether at some point.